src/main/java/uk/gov/digital/ho/hocs/templates/TemplateService.java

```diff
@@ -79,15 +76,18 @@ public TemplateResult buildTemplate(UUID caseUUID, UUID templateUUID) {
         return new TemplateResult(caseDetails.getReference(), generatedTemplate);
     }
 
-
-    private TeamDto getTeamDetails(UUID caseUUID, CaseDataDto caseDetails) {
+    private TeamDto getPrivateOfficeTeamDetails(UUID caseUUID, CaseDataDto caseDetails) {
+        List<String> privateOfficeCaseTypes = List.of("MIN", "DTEN");
```

Comment:
Can we pull this list from somewhere instead of magic numbering it? (I have no idea)

Reply:
I've looked at using `@Value` and storing this in the `application.properties` file but it really doesn't want to pull from the file (in the tests anyway). Could we put this into a ticket to go look back as part of the wider templates refactor?

Reply:
Would it be more consistent to put the application specific data (which types of case have which office types) in hocs-data and avoid putting such specific information in the services? Although that would be a much bigger piece of work, so maybe it's an okay compromise to do something like put it in a named constant.

Reply:
This now pulled from info-service and our data repo through API endpoint - `/caseType/{type}/stages`.